static/js/compose.vue

```diff
@@ -201,6 +201,9 @@
       submit: function(){
         var duration = moment.duration(this.duration_data.duration, 'seconds');
         var duration_string = '';
+        if(duration.days() > 0){
+            duration_string = duration.days() + ' days ' + duration_string;
```

Comment:
should there be a comma after days to be consistent with hours, minutes,?

Reply:
done